src/components/modules/ui.ts

```diff
@@ -340,7 +340,7 @@ export default class UI extends Module<UINodes> {
       this.documentKeydown(event);
     }, true);
 
-    this.readOnlyMutableListeners.on(document, 'click', (event: MouseEvent) => {
+    this.readOnlyMutableListeners.on(document, 'mousedown', (event: MouseEvent) => {
```

Comment:
is it break something?

Reply:
No, I went thru documentClicked method and there is no dependencies on click phase